tests/unit_tests/local_server.py

```
@@ -25,6 +29,26 @@ def DeleteShares(self, request, context):
         return res
 
     def ExecuteComputation(self, request, context):
+        if request.token == "QMPCJobError":
```

I would like you to fix it, because it is far from practical to judge by TOKEN.
I thought it would be better to return an error with an argument that would cause that error.
For example
```Python
if len(request.arg.src) > 0 \
        and request.arg.src[0] == 1000000000: # QMPCJobError

if len(request.table.dataIds) > 0 \
        and request.table.dataIds[0] == "UnregisteredDataId": # QMPCServerError
```

---

Thank you!  
Your review is helpful because "I" told that there is also a way to use tokens.